src/config/dynamic-theme-fixes.config

```diff
@@ -5257,6 +5257,22 @@ INVERT
 
 ================================
 
+circuits.vcn.com
+
+INVERT
+img
+
+CSS
+select {
+    background-color: #121212 !important;
```

Comment:
Please use `${initial_color} !important` or `var(--darkreader-neutral-background) !important` to make theme applied properly. See more info here https://github.com/darkreader/darkreader/blob/main/CONTRIBUTING.md

Reply:
So I changed this to use variables instead of hardcoded colors and added some styling so that you can see the arrows for a select menu in certain cases. The colors look the same as far as I can tell, but the dropdowns on some selects look better now:
normal select before and after:
![image](https://github.com/darkreader/darkreader/assets/77451223/bd8ab880-7ba0-461d-a676-26b616cca573)

special case select before:
![image](https://github.com/darkreader/darkreader/assets/77451223/3b80d28a-bdd0-487b-9661-d498f2723328)
and after: 
![image](https://github.com/darkreader/darkreader/assets/77451223/24b009e0-9b41-4148-b47c-71c102b8ead3)

notice that the arrow is visible in the dropdown.